src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/197401910-d199c8cd-b766-4698-a52a-9635720b1f09.png)
**Not corrected.**
Make Random **class-level** variable. Now it is **local** variable.